changes/259.added

```diff
@@ -0,0 +1 @@
+Added screenshots to docs/images.
```

Comment:
Should these files be removed now that they're reflected in `docs/release_notes/version_1.0.md`? Normally I'd expect `towncrier` to do that for us.

Reply:
towncrier, in this repository, it's only checking for this file per PR, but it's not doing anything for the release notes (I may be missing something)